specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-08-02-preview/managedClusters.json

```diff
@@ -3483,6 +3486,10 @@
           },
           "title": "The fully qualified resource ID of the Dedicated Host Group to provision virtual machines from, used only in creation scenario and not allowed to changed once set.",
           "description": "This is of the form: /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/hostGroups/{hostGroupName}. For more information see [Azure dedicated hosts](https://docs.microsoft.com/azure/virtual-machines/dedicated-hosts)."
+        },
+        "windowsProfile": {
+          "$ref": "#/definitions/AgentPoolWindowsProfile",
+          "description": "The Windows node's specific profile."
```

Comment:
The Windows agent pool's specific profile?

Reply:
Since WindowsProfile in cluster level is also applied in Windows nodes.

Reply:
Same in line 7065

Reply:
@ShiqianTao Added two more comments for the same line in the same conversation.

Reply:
Updated them. Thanks for your guidance @AbelHu